src/components/CustomerQuota/CheckoutSuccess/CheckoutSuccessCard.tsx

```diff
@@ -166,8 +166,18 @@ export const CheckoutSuccessCard: FunctionComponent<CheckoutSuccessCard> = ({
     !!quotaResponse?.globalQuota &&
     !!sortedTransactions &&
     sortedTransactions.length > 0 &&
+    /**
+     * We only display global limit messages if there is only one global quota,
+     * since we have not catered for showing global limit messages for multiple
+     * categories.
+     */
+    quotaResponse.globalQuota.length === 1 &&
     !!getProduct(sortedTransactions[0].category)?.quantity.usage;
 
+  const globalQuota = showGlobalQuota
```

Comment:
Hmmm thinking how we might be able to improve the naming to reflect that this is different from the list of global quota items `quoteResponse.globalQuota`. `firstGlobalQuota`?

Reply:
`firstGlobalQuota` sounds good!